packages/lab/src/dropdown-next/DropdownNext.css

```diff
@@ -1,108 +1,98 @@
-.saltDropdownNext-button {
-  --dropdownNext-borderColor: var(--salt-editable-borderColor);
-  --dropdownNext-borderStyle: var(--salt-editable-borderStyle);
-  --dropdownNext-borderWidth: var(--salt-size-border);
+.saltDropdownNext {
+  --dropdown-next-borderColor: var(--salt-editable-borderColor);
```

Comment:
everywhere else we use camel case, why are you changing it here?

Reply:
I think when I first tried to do it I had weird linting issues but it should be consistent so I've updated it